lib/Artwork.php

```diff
@@ -168,6 +190,19 @@ public function Create(): void{
 		}
 	}
 
+	/**
+	 * @throws \Exceptions\ValidationException
+	 */
+	public function Save(): void{
```

Comment:
Suggestion: the name of this method could be more specific -- `SaveStatus` or something that indicates that it's only updating one field.

Reply:
I think the idea is that you could modify multiple properties on an object, then call `Save()` once (which would be just one database `UPDATE`).

[`NewsletterSubscription`](https://github.com/standardebooks/web/blob/5ea795881c86ca70b1292aba7185ef9a6928a555/lib/NewsletterSubscription.php#L73) can update three properties. [Here](https://github.com/standardebooks/web/blob/5ea795881c86ca70b1292aba7185ef9a6928a555/www/newsletter/subscriptions/post.php#L84) is an example of getting a NewsletterSubscription object, modifying it, and calling `Save()`.

Reply:
Fair enough - we can change the method if there are more properties that need to be updated.